games-roguelike/FTL-gog/FTL-gog-1.6.9.25330.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop unpacker
+
+DESCRIPTION="Top-down roguelike space ship simulator"
+HOMEPAGE="https://www.gog.com/game/faster_than_light"
+SRC_URI="ftl_advanced_edition_${PV//./_}.sh"
+RESTRICT="bindist fetch mirror strip test"
+
+LICENSE="GOG-EULA"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE=""
+
+DEPEND="app-arch/unzip"
+RDEPEND="media-libs/freetype
+	media-libs/libsdl[X,sound,opengl,video]
+	media-libs/libpng
+	sys-libs/zlib
+	virtual/opengl"
+
+S="${WORKDIR}/data/noarch"
+
+pkg_nofetch() {
+	elog "Please buy and download ${SRC_URI} from"
+	elog "https://www.gog.com/game/faster_than_light"
+}
+
+src_unpack() {
+	unpack_zip "${DISTDIR}/${A}"
+}
+
+src_prepare() {
+	default
+	rm -r support/{xdg*,*.{sh,txt}} || die
+	if ! use x86; then
+		rm game/data/FTL.x86 || die
+	fi
+	if ! use amd64; then
+		rm game/data/FTL.amd64 || die
+	fi
+
+	sed -i start.sh -e '/chmod/d'
```

Comment:
Please append `|| die` here.

Reply:
Good catch. Nobody noticed that when that line was introduced as a patch for the previous version's ebuild.